airbyte-ci/connectors/pipelines/pipelines/gradle.py

```diff
@@ -90,7 +90,7 @@ async def _run(self) -> StepResult:
             "findutils",  # gradle requires xargs, which is shipped in findutils.
             "jq",  # required by :airbyte-connector-test-harnesses:acceptance-test-harness to inspect docker images.
             "npm",  # required by :format.
-            "pip",  # required by :format.
+            "python3.11-pip",  # required by :format.
```

Comment:
❗some tools (ci_credentials, pipelines) cant currently support python3.11, this wont be an issue here but is an edge case. is python 3.10 available instead?

Reply:
For some reason that I don't understand, no. If it comes to that, it'll probably be simpler to change the base container to something other than amazoncorretto. I only picked it because openjdk was no longer maintained and amazoncorretto has LTS.